content/pt-br/distributed-systems.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Sistemas Distribuídas
+status: Completed
+category: conceito
+tags: ["arquitetura", "", ""]
+---
+
+## O que é
+
+Um sistema distribuído é uma coleção de elementos de computação autônomos conectados por uma rede que aparece para os usuários como um único sistema coerente. Geralmente referidos como [nós](/pt-br/nodes/), esses componentes podem ser dispositivos de hardware (por exemplo, computadores, telefones móveis) ou processos de software. Os nós são programados para alcançar um objetivo comum e, para colaborar, eles trocam mensagens pela rede.
+
+## Problema relacionado     
+
+Inúmeras aplicações modernas hoje são tão grandes que precisariam de supercomputadores para operar. Pense no Gmail ou Netflix. Nenhum computador único é suficientemente poderoso para hospedar toda a aplicação. Ao conectar vários computadores, o poder computacional se torna praticamente ilimitado. Sem a computação distribuída, muitas das aplicações das quais dependemos hoje não seriam possíveis.
+
+Tradicionalmente, os sistemas seriam escalados verticalmente. Isso significa adicionar mais CPU ou memória a uma máquina individual. A [escalabilidade vertical](/vertical-scaling/) é demorada, requer tempo de inatividade e atinge seu limite rapidamente.
```

Comment:
Add link to scale (/pt-br/scalability/)
Add /pt-br/ on vertical scaling link

Reply:
updated